lib/components/viewers/route-row.tsx

```diff
@@ -37,6 +37,7 @@ export const StyledRouteRow = styled.li`
 // Route Row Button sits invisible on top of the route name and info.
 export const RouteRowLink = styled(Link)`
   align-items: center;
+  border-radius: 4px;
```

Comment:
Any chance we can bump this up to maybe 10? Just to be in line with some of the rest of our UI

Reply:
Increassed to 10px in https://github.com/opentripplanner/otp-react-redux/pull/1216/commits/ec9b5527f61e24d4c9a952b5ab9d3754e0582d7d, so it is like the Nearby View.